source/onboard/ssl-client-certificate.rst

```diff
@@ -10,8 +10,9 @@ Before you begin, follow the `official guides to install Mattermost </guides/adm
 
 For the purposes of this guide, the Mattermost server domain name is ``example.mattermost.com``, and the user account is ``mmuser`` with email ``mmuser@mattermost.com`` and password ``mmuser-password``.
 
-.. note::
-  Generating the client certificates in this section is optional if you have already generated them before.
+.. important::
+  - Generating the client certificates in this section is optional if you have already generated them before.
+  - If your production Mattermost Server instance **Site URL** is ``HTTP`` or ``localhost``, we strongly recommend configuring an SSL certificate (or a self-signed certificate) for security, privacy, compliance, as well as to avoid browser limitations that can prevent Mattermost product features from working that copy data using the user's local clipboard, including `sharing links in messages </channels/share-links.html>`__ and `inviting new members to teams </welcome/about-teams.html#send-a-team-invite-link>`__. 
```

Comment:
I would just remove this part from this sentence:
`If your production Mattermost Server instance **Site URL** is ``HTTP`` or ``localhost``,`